### DISCUSSION TITLE
Welcome to tibia_terminator Discussions!

### DISCUSSION THREAD

**Discussion by marcel-valdez** [marcel-valdez/tibia_terminator#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Dettao** [comment]
Could you explain me how to do this part please? i am using ubuntu 20.04.
"## How to bring it up




1. Set environment variables"
